### ISSUE TITLE
add bridgedb config line parser

### ISSUE THREAD

**Issue by n8fr8** [guardianproject/AndroidPluggableTransports#13]

**Comment by eighthave** [comment]
Does _Obfs4Transport.setPropertiesFromBridgeString()_ qualify?  I.e. https://github.com/guardianproject/AndroidPluggableTransports/blob/7b8b6f44edc411b6ec02349816da38fa3df93670/APTDispatchLibrary-MeekObfs4-Full/src/main/java/info/pluggabletransports/dispatch/transports/Obfs4Transport.java#L310

Or were you thinking of something different?  I was thinking of adding standard parsing methods for Bridge URLs here as well. https://trac.torproject.org/projects/tor/ticket/15035

**Comment by n8fr8** [comment]
yeah I think this is done WRT to the original request, though I need to make sure that method is also in the LEGACY module that is currently working.
